### PR TITLE
✍️ RTL support, 3+ RTL languages, "Theme" translation fix in SettingsSidebar (🐛)

### PR DESCRIPTION
**Pull request by Cs4K1Sr4C**

### CODE REVIEW

**Review by Cs4K1Sr4C — COMMENTED** [review]
Almost perfect...

**Comment by Cs4K1Sr4C** [comment]
{ code: 'br', name: 'Português (Brasil)', flag: '🇧🇷' },
  { code: 'ar', name: 'Arabian', flag: '🇦🇪' },
  { code: 'de', name: 'Deutsch', flag: '🇩🇪' },
  { code: 'es', name: 'Español', flag: '🇪🇸' },
  { code: 'fr', name: 'Français', flag: '🇫🇷' },
  { code: 'hi', name: 'हिन्दी', flag: '🇮🇳' },
  { code: 'hu', name: 'Magyar', flag: '🇭🇺' },
  { code: 'ja', name: '日本語', flag: '🇯🇵' },
  { code: 'ru', name: 'Русский', flag: '🇷🇺' },
  { code: 'th', name: 'ไทย', flag: '🇹🇭' },
  { code: 'ko', name: '한국어', flag: '🇰🇷' },
  { code: 'pt', name: 'Português', flag: '🇵🇹' },
  { code: 'id', name: 'Indonesia', flag: '🇮🇩' },
  { code: 'it', name: 'Italiano', flag: '🇮🇹' },
  { code: 'uk', name: 'Українська', flag: '🇺🇦' },
  { code: 'zh', name: '中文', flag: '🇨🇳' },
  { code: 'vi', name: 'Tiếng Việt', flag: '🇻🇳' },
  { code: 'au', name: 'Australian', flag: '🇦🇺' },
  { code: 'gb', name: 'British', flag: '🇬🇧' },
  { code: 'lt', name: 'Lietuvių', flag: '🇱🇹' },
  { code: 'cs', name: 'Čeština', flag: '🇨🇿' },
  { code: 'el', name: 'Ελληλαηλοή', flag: '🇬🇷' },
  { code: 'sv', name: 'Svenska', flag: '🇸🇪' },
  { code: 'da', name: 'Dansk', flag: '🇩🇰' },
  { code: 'fi', name: 'Suomi', flag: '🇫🇮' },
  { code: 'no', name: 'Norsk', flag: '🇳🇴' },
  { code: 'tr', name: 'TÜRKİSH', flag: '🇹🇷' },
  { code: 'pl', name: 'Polski', flag: '🇵🇱' },
  { code: 'zhtw', name: '中文(台灣)', flag: '🇹🇼' },
  { code: 'bg', name: 'бълежи', flag: '🇧🇬' },
  { code: 'nl', name: 'Nederlands', flag: '🇳🇱' },
  { code: 'sk', name: 'Slovenčina', flag: '🇸🇰' },
  { code: 'ro', name: 'Română', flag: '🇷🇴' },
  { code: 'lv', name: 'Latviešu', flag: '🇱🇻' },
  { code: 'et', name: 'Eesti', flag: '🇪🇪' },
  { code: 'hr', name: 'Hrvatski', flag: '🇭🇷' },
  { code: 'sl', name: 'Slovenščina', flag: '🇸🇮' },
  { code: 'el', name: 'Ελληνικά', flag: '🇬🇷' },
  { code: 'ru', name: 'Русский', flag: '🇷🇺' },
  { code: 'uk', name: 'Українська', flag: '🇺🇦' },
  { code: 'sr', name: 'Српски', flag: '🇷🇸' },
  { code: 'he', name: 'עברית', flag: '🇮🇱' },
  { code: 'ar', name: 'العربية', flag: '🇸🇦' },
  { code: 'fa', name: 'فارسی', flag: '🇮🇷' },
  { code: 'ur', name: 'اردو', flag: '🇵🇰' },
  { code: 'hi', name: 'हिन्दी', flag: '🇮🇳' },
  { code: 'bn', name: 'বাংলা', flag: '🇧🇩' },
  { code: 'gu', name: 'ગુજરાતી', flag: '🇮🇳' },
  { code: 'ta', name: 'தமிழ்', flag: '🇮🇳' },
  { code: 'te', name: 'తెలుగు', flag: '🇮🇳' },
  { code: 'kn', name: 'ಕನ್ನಡ', flag: '🇮🇳' },
  { code: 'ml', name: 'മലയാളം', flag: '🇮🇳' }

**Review by Cs4K1Sr4C — COMMENTED** [review]
I'll fix the zh-tw translation in the next PR.

**Comment by miurla** [comment]
Deployment errors in vercel

```
[19:12:54.257] > babyagi-ui@0.1.0 build
[19:12:54.257] > next build
[19:12:54.257] 
[19:12:54.858] 
[19:12:54.859] > Build error occurred
[19:12:54.862] Error: Specified i18n.defaultLocale should be included in i18n.locales.
[19:12:54.862] See more info here: https://nextjs.org/docs/messages/invalid-i18n-config
[19:12:54.862]     at assignDefaults (/vercel/path0/node_modules/next/dist/server/config.js:555:19)
[19:12:54.863]     at Object.loadConfig [as default] (/vercel/path0/node_modules/next/dist/server/config.js:128:32)
[19:12:54.863]     at async Span.traceAsyncFn (/vercel/path0/node_modules/next/dist/trace/trace.js:79:20)
[19:12:54.863]     at async /vercel/path0/node_modules/next/dist/build/index.js:78:28
[19:12:54.863]     at async Span.traceAsyncFn (/vercel/path0/node_modules/next/dist/trace/trace.js:79:20)
[19:12:54.864]     at async Object.build [as default] (/vercel/path0/node_modules/next/dist/build/index.js:73:29)
[19:12:54.898] Error: Command "npm run build" exited with 1
[19:12:55.074] Deployment completed
[19:12:55.033] BUILD_UTILS_SPAWN_1: Command "npm run build" exited with 1
```

File: src/utils/languages.ts [hunk 1]
```diff
@@ -34,9 +35,31 @@ export const availableLanguages: Language[] = [
...
+  { code: 'nl', name: 'Nederlands', flag: '🇳🇱' },
+  { code: 'sk', name: 'Slovenčina', flag: '🇸🇰' },
+  { code: 'ro', name: 'Română', flag: '🇷🇴' },
+  { code: 'lv', name: 'Latviešu', flag: '🇱🇻' },
+  { code: 'et', name: 'Eesti', flag: '🇪🇪' },
+  { code: 'hr', name: 'Hrvatski', flag: '🇭🇷' },
+  { code: 'sl', name: 'Slovenščina', flag: '🇸🇮' },
+  { code: 'el', name: 'Ελληνικά', flag: '🇬🇷' },
+  { code: 'ru', name: 'Русский', flag: '🇷🇺' },
+  { code: 'uk', name: 'Українська', flag: '🇺🇦' },
```
Comment (miurla) [inline src/utils/languages.ts:50]: duplication

https://github.com/miurla/babyagi-ui/pull/57/files#diff-6703b91dc8242fb5425dfa1a282a7ea5c9a9bc92cd8f453a61aa06dc6b2eb5b0R25

**Review by miurla — COMMENTED** [review]
RTL is amazing!
There was a duplicate in availableLanguages, please just fix that!

**Comment by miurla** [comment]
uk 👀 

https://github.com/miurla/babyagi-ui/pull/57#discussion_r1200463624

File: src/utils/languages.ts (hunk 1 above)
Comment (Cs4K1Sr4C) [reply src/utils/languages.ts:48]: Ohh, I found an other one not uk 😅 It duplicated because I force pushed it.

Comment (miurla) [reply src/utils/languages.ts:47]: Thx!

**Review by miurla — APPROVED** [review]
Thanks!
RTL Support is amazing! 🚀

**Pull request merged**